### ISSUE TITLE
Jobber commands are timing out intermittently

### ISSUE THREAD

**Issue by ttrading** [dshearer/jobber#280]
My commands such as "jobber list" time out intermittently on my system. Has anyone run into this issue? I read about a potential solution to set a timeout value but I cannot find any documentation or reference about it.

Thanks

**Comment by dshearer** [comment]
Do you think you could be seeing #283 ?

**Comment by ttrading** [comment]
I am not sure. I will try to replicate and update the ticket. 

If it is not, is it possible to change the default timeout from 5 seconds yet?

Thanks

**Comment by dshearer** [comment]
No, it's not possible to change the timeout.  It really shouldn't be timing out, though, so I don't think increasing the timeout would help.
